src/win/path_util.cc

```diff
@@ -4,25 +4,28 @@
  * Copyright (c) 2018, Microsoft Corporation (MIT License).
  */
 
-#include <nan.h>
 #include <Shlwapi.h> // PathCombine
 
 #include "path_util.h"
 
 namespace path_util {
 
-const wchar_t* to_wstring(const Nan::Utf8String& str) {
-  const char *bytes = *str;
-  int sizeOfStr = MultiByteToWideChar(CP_UTF8, 0, bytes, -1, NULL, 0);
-  if (sizeOfStr <= 0) {
-    return L"";
+std::wstring to_wstring(const Napi::String& str) {
+  const std::u16string & u16 = str.Utf16Value();
+  return std::wstring(u16.begin(), u16.end());
+}
+
+std::string wstring_to_string(const std::wstring &wide_string) {
```

Comment:
The `Napi::Error::New` only allow utf-8 `std::string`, so we need this to convert.

<!-- https://stackoverflow.com/a/69410299/11338291 -->